boto3/docs/method.py

```diff
@@ -46,7 +46,8 @@ def document_model_driven_resource_method(
         return_description = 'A %s resource' % (
             resource_action_model.resource.type)
 
-        if resource_action_model.path and '[]' in resource_action_model.path:
+        if (resource_action_model.resource.path and
```

Comment:
So I think this logic that I originally put in place only accounts for a subset of the all of the possible occasions where a list of resources can be returned. After talking with @trevorrowe, the only true way to tell if a list or resources is returned is if the returned resource definition has at least one identifier that has a ``[]`` in its path. It would be cool if we can update the logic to that because your fix currently will not fix some other edge cases such as any of the create_tags actions. Here is the model https://github.com/boto/boto3/blob/develop/boto3/data/ec2/2014-10-01/resources-1.json#L528 and this is what the docs output: http://boto3.readthedocs.org/en/latest/reference/services/ec2.html#EC2.Image.create_tags which is incorrect.